### ISSUE TITLE
magc: viewport horizontally flipped compared to smartsem

### ISSUE THREAD

**Issue by templiert** [SBEMimage/SBEMimage#23]
After wafer calibration, the coordinate systems are correct (right click in viewport on a section visible in the wafer image moves the stage correctly to that section). 

A cosmetic issue remains: the viewport is horizontally flipped compared to the smartsem view

![viewport_reflection_1](https://user-images.githubusercontent.com/34257249/64716857-75b4b880-d4c3-11e9-9cd2-72b40ed636b2.jpg)

![smartsem_reflection_1](https://user-images.githubusercontent.com/34257249/64716866-7c433000-d4c3-11e9-982a-93b9487cfd93.jpg)

**Comment by btitze** [comment]
Another issue with the rotation angle (which may be related):
I've noticed that you are using 
self.sem.set_scan_rotation(360 - theta)
But I'm using 
self.sem.set_scan_rotation(theta)

and it seems to work on our Merlin.

**Comment by templiert** [comment]
> I've noticed that you are using
> self.sem.set_scan_rotation(360 - theta)

No, I'm using self.sem.set_scan_rotation(theta) like you [see here](https://github.com/SBEMimage/SBEMimage/blob/magc-integration/src/stack_acquisition.py#L1697)

(I believe I had originally set (360-theta) before the implementation of the rotations because I had been anticipating an issue that finally did not occur)

**Comment by templiert** [comment]
editing is currently broken, I skipped a bracket:

No, I'm using self.sem.set_scan_rotation(theta) like you [(see here)](https://github.com/SBEMimage/SBEMimage/blob/magc-integration/src/stack_acquisition.py#L1697)

**Comment by btitze** [comment]
Oops, I meant the opposite. The lines got swapped:

I am actually using `self.sem.set_scan_rotation(360 - theta)`

We should look into that. I don't understand why it should be different unless it's a machine-specific issue.

**Comment by folterj** [comment]
Archiving this as no longer an issue.
